robottelo/cli/imprt.py

```diff
@@ -0,0 +1,341 @@
+            path = cls.prepare_import_data()
+            options = {
+                u'csv-file': '{0}/exports/system-profiles.csv'.format(path),
+            }
+        elif 'csv-file' not in options or options['csv-file'] is None:
+            path = cls.prepare_import_data()
+            options.update({
+                u'csv-file': '{0}/exports/system-profiles.csv'.format(path),
+            })
+
+        result = cls.execute(
+            cls._construct_command(options),
+            output_format=''
+        )
+
+        return result
+
+    @classmethod
+    def content_view(cls, options=None):
+        """Create Content Views based on local/cloned Channels
+        (from spacewalk-export-channels)."""
+
+        cls.command_sub = 'content-view'
+
+        if options is None:
+            path = cls.prepare_import_data()
+            options = {
+                u'csv-file': '{0}/exports/channels.csv'.format(path),
+            }
+        elif 'csv-file' not in options or options['csv-file'] is None:
+            path = cls.prepare_import_data()
+            options.update({
+                u'csv-file': '{0}/exports/channels.csv'.format(path),
+            })
+
+        result = cls.execute(
+            cls._construct_command(options),
+            output_format=''
+        )
+
+        return result
+
+    @classmethod
+    def repository(cls, options=None):
+        """Import repositories (from spacewalk-report repositories)."""
+
+        cls.command_sub = 'repository'
+
+        if options is None:
+            path = cls.prepare_import_data()
+            options = {
+                u'csv-file': '{0}/exports/repositories.csv'.format(path),
+            }
+        elif 'csv-file' not in options or options['csv-file'] is None:
+            path = cls.prepare_import_data()
+            options.update({
+                u'csv-file': '{0}/exports/repositories.csv'.format(path),
+            })
+
+        result = cls.execute(
+            cls._construct_command(options),
+            output_format=''
+        )
+
+        return result
+
+    @classmethod
+    def repository_enable(cls, options=None):
+        """Enable any Red Hat repositories accessible to any Organization
+        (from spacewalk-report channels)."""
+
+        cls.command_sub = 'repository-enable'
+
+        if options is None:
+            path = cls.prepare_import_data()
+            options = {
+                u'csv-file': '{0}/exports/channels.csv'.format(path),
+            }
+        elif 'csv-file' not in options or options['csv-file'] is None:
+            path = cls.prepare_import_data()
+            options.update({
+                u'csv-file': '{0}/exports/channels.csv'.format(path),
+            })
+
+        result = cls.execute(
+            cls._construct_command(options),
+            output_format=''
+        )
+
+        return result
+
+    @classmethod
+    def template_snippet(cls, options=None):
+        """Import template snippets
```

Comment:
`"""Import template snippets (from spacewalk-report kickstart-scripts)."""` if does not fits in one line:

```python
`"""Import template snippets (from spacewalk-report
kickstart-scripts).

"""`